src/components/App/index.js

```diff
@@ -165,14 +169,22 @@ class App_ extends React.Component<AppProps, AppState> {
       })
   }
 
+  checkIfAboutPage() {
+    return window.location.href.includes('/about')
+  }
+
   render() {
     const decoratedToggleAboutPage = () => {
-      const query = window.location.search
+      // this conditional seems backward, because it is. this fn
+      // operates from the "before" state -- if `showAboutPage`
+      // is true, we need to transition to a state wherein
+      // `showAboutPage` is false.
 
-      if (!this.props.showAboutPage) {
-        window.history.pushState({}, '', `/about/${query}`)
+      if (this.props.showAboutPage) {
+        const { id } = this.props.focusedSubject
+        window.history.pushState({}, '', `/?subject=${id}`)
```

Comment:
If this actually does work in all browsers, then we should remove the try/catch that we have around pushState in App/index.js.